src/collector/parsers/sass.ts

```diff
@@ -100,7 +100,7 @@ function extractArgument(declarationNode: any) {
     const valueNode = find(declarationNode, { type: 'value' });
 
     return {
-        name: nameNode.value,
+        name: (nameNode as any).value,
```

Comment:
Can we bind the type on initialization instead?
e.g. `const nameNode = find(variableNode, { type: 'ident' }) as any;`